### ISSUE TITLE
Be able to run on JDK9

### ISSUE THREAD

**Issue by ktoso** [ThoughtWorksInc/sbt-api-mappings#8]
In Akka we're trying to release for JDK9 already and are working on the entire plugin ecosystem to allow this. I found that there's a missing case in this plugin too which makes it not usable with JDK9:

```

[error] scala.MatchError: 9 (of class java.lang.String)
[error] 	at com.thoughtworks.sbtApiMappings.BootstrapApiMappings$.$anonfun$globalSettings$1(BootstrapApiMappings.scala:28)
[error] 	at sbt.internal.util.Init$Value.$anonfun$apply$3(Settings.scala:804)
```

**Comment by Atry** [comment]
An workaround:
``` sbt
disablePlugins(BootstrapApiMappings) // For JDK 9
```

**Comment by Atry** [comment]
Fixed in #12

**Comment by ashawley** [comment]
More changes related to Java 9 added in #13
